conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/openapi/OpenApiDoc.scala

```diff
@@ -40,6 +43,18 @@ object OpenApiDoc
     bitcoinOutputsEndpoint,
     bitcoinAccountsEndpoint,
     bitcoinAccountByAddressEndpoint,
+    ethereumBlocksEndpoint,
+    ethereumBlocksHeadEndpoint,
+    ethereumBlockByHashEndpoint,
+    ethereumTransactionsEndpoint,
+    ethereumTransactionByHashEndpoint,
+    ethereumLogsEndpoint,
+    quorumBlocksEndpoint,
+    quorumBlocksHeadEndpoint,
+    quorumBlockByHashEndpoint,
+    quorumTransactionsEndpoint,
+    quorumTransactionByHashEndpoint,
+    quorumLogsEndpoint,
```

Comment:
What would you think about each platform providing a method that collects all endpoints for each?
The end result should look something like

```scala
openApi(Info...) {
  queryEndpoint ::
  platformsEndpoint ::
  entitiesEndpoint ::
  tezosEndpoints ++
  bitcoinEndpoints ++
  ethereumEndpoints ++
  quorumEndoints : _*
}
```

Reply:
I'm pretty sure this is technically not possible due to very weird types returned by endpoints4j library. I tried that already, when I introduced second blockchain. I don't want to put compiler errors here, but if you want you can quickly validate your solution. Just try to add: `Seq(bitcoinOutputsEndpoint, bitcoinAccountsEndpoint): _*` at the end. That's pretty sad. Maybe it is solved in the latest endpoints4j library, but I stuck couple days ago on parsing arguments for Tezos Endpoint. There are many of them, where endpoints4j supports up to 5 out of the box, so we need to solve it first. Either way, it's kind of mess there (in this lib, with these types).